pkg/pgmodel/sql_ingest.go

```diff
@@ -28,9 +28,10 @@ const (
 )
 
 type Cfg struct {
-	AsyncAcks       bool
-	ReportInterval  int
-	SeriesCacheSize uint64
+	AsyncAcks          bool
+	ReportInterval     int
+	SeriesCacheSize    uint64
+	ConnectionsPerProc int
```

Comment:
I think it would be better to make this `IngestorConnections` or `copierConnections` and have all those calculations be done in pgClient. This would mean we wouldn't have to calculate maxProcs in the pgmodel code etc